src/localization/german/kmpmp_companies_l_german.yml

```diff
@@ -12,4 +12,8 @@
 
  company_basic_furniture: "Möbelwerke"
  company_basic_furniture_dynamic_name_tag_singular: "Möbel"
- company_basic_furniture_dynamic_name_tag_plural: "Möbel"
\ No newline at end of file
+ company_basic_furniture_dynamic_name_tag_plural: "Möbel"
+ company_Korea_Tungsten_Company: "Korea Tungsten Company"
+ company_Imperial_Arsenal: "Imperial Arsenal"
+ company_Genda_Shigyo: "Geda Shigyo"
```

Comment:
german translation if possible - at least Korea company can be translated, in english in should be "Korean" instead of "Korea"?

Reply:
done